### ISSUE TITLE
UI shows own email when viewing affiliation requests for an organization

### ISSUE THREAD

**Issue by vegu** [peeringdb/peeringdb#1250]
**Describe the bug**

When viewing organization affiliation requests, the UI current displays your own email address instead of the email address attached to the affiliation request.

This makes it impossible to reasonably confirm the requesting user's identity.

**To Reproduce**
1. Go to /org/xxx
2. click Users tab
3. any row in the **Users requesting affiliation** list will show your own email address in the email column

**Comment by arnoldnipper** [comment]
Routed through to 3a as confirmed to be a bug by a developer

**Comment by ccaputo** [comment]
I ran into this today. Verified it on current beta and production. At first I thought my account was at risk or someone had used my email. Would be lovely to get this fixed.

Here is a demonstration on beta with https://beta.peeringdb.com/org/550 . Note that the SIX user does not have my email, nor are there two accounts with the same email address:

![Screen Shot 2022-10-24 at 7 52 10 AM](https://user-images.githubusercontent.com/1417833/197556526-73a35e2e-c4ce-4e31-a1f0-eee7b8821b8e.png)
